Dockerfile

```diff
@@ -43,9 +43,10 @@ RUN apk add --update \
   heimdal \
   && rm -rf /var/cache/apk/*
 
-#wheel needed by ruamel.yaml for some reason
-RUN pip3 install -U wheel setuptools pip
-RUN pip3 install cwltool
+# wheel needed by ruamel.yaml for some reason
+# https://stackoverflow.com/a/75722775
+RUN pip3 install --break-system-packages -U wheel setuptools pip
+RUN pip3 install --break-system-packages cwltool
```

Comment:
Maybe use pipx instead?

Reply:
We can do that too, but then we will have to modify another step above to install it, with `sudo apt install pipx` I think. Or use another docker multi-stage build image to get python with some virtual env (mamba/pip/etc). Maybe we can create an issue to check which option would work best.

Reply:
I think adding `pipx` to the apt-get install will work easily